frontend/src/components/starmap/StarMap.vue

```diff
@@ -2,6 +2,12 @@
   <div class="canvas-container">
     <canvas ref="canvas"></canvas>
   </div>
+  <HoverInfo
+    :data="hoveredStar.data"
+    :x="hoveredStar.x"
+    :y="hoveredStar.y"
```

Comment:
You can certainly refactor this:
```suggestion
    :hoveredStar="hoveredStar"
```